--- libpubnub-cpp/Makefile
+++ libpubnub-cpp/Makefile
@@ -31,6 +31,5 @@ endif
 	ln -s -f libpubnub-cpp.so.0.1 $(DESTDIR)$(LIBDIR)/libpubnub-cpp.so.0
 	ln -s -f libpubnub-cpp.so.0.1 $(DESTDIR)$(LIBDIR)/libpubnub-cpp.so
 	$(INSTALL) -D -m 0644 libpubnub-cpp.pc $(DESTDIR)$(LIBDIR)/pkgconfig/libpubnub-cpp.pc
-	ldconfig
 
 -include ../Makefile.lib
--- libpubnub/Makefile
+++ libpubnub/Makefile
@@ -32,6 +32,5 @@ endif
 	ln -s -f libpubnub.so.0.1 $(DESTDIR)$(LIBDIR)/libpubnub.so.0
 	ln -s -f libpubnub.so.0.1 $(DESTDIR)$(LIBDIR)/libpubnub.so
 	$(INSTALL) -D -m 0644 libpubnub.pc $(DESTDIR)$(LIBDIR)/pkgconfig/libpubnub.pc
-	ldconfig
 
 -include ../Makefile.lib
