--- lein-bin
+++ lein-bin
@@ -31,7 +31,7 @@
 # User init
 export LEIN_HOME="${LEIN_HOME:-"$HOME/.lein"}"
 
-for f in "$LEIN_HOME/leinrc" ".leinrc"; do
+for f in "/etc/leinrc" "$HOME/.leinrc"; do
     if [ -e "$f" ]; then
         source "$f"
     fi
@@ -43,7 +43,7 @@
 # If you're not using an uberjar you'll need to list each dependency
 # and add them individually to the classpath/bootclasspath as well.
 
-LEIN_JAR=/usr/share/java/leiningen-$LEIN_VERSION-standalone.jar
+LEIN_JAR=/opt/leiningen-bin-0/lib/leiningen-$LEIN_VERSION-standalone.jar
 
 # Do not use installed leiningen jar during self-compilation
 if ! { [ "$1" = "compile" ] &&
